src/Templates/SignUp/SignUp.js

```diff
@@ -9,29 +9,21 @@ const SignUp = () => {
   return (
     <>
       <GradientLogoFrame
```

Comment:
if its a gradientLogofrem that why we should pass gradient as true change the propose or else change the file name